release-please-config.json

```diff
@@ -27,11 +27,17 @@
       "changelog-path": "CHANGELOG.md",
       "initial-version": "0.1.0"
     },
-    "openfeature-provider-java": {
+    "openfeature-provider/java": {
       "path": "openfeature-provider/java",
       "release-type": "rust",
       "changelog-path": "CHANGELOG.md",
-      "extra-files": ["pom.xml", "version.txt"]
+      "extra-files": ["pom.xml"]
+    },
+    "openfeature-provider/js": {
+      "path": "openfeature-provider/js",
+      "release-type": "rust",
+      "changelog-path": "CHANGELOG.md",
+      "extra-files": ["package.json"]
```

Comment:
Package name and path must match for release please to update the package.json. For consistency, I changed the package name of Java as well